.ci.yaml

```diff
@@ -25,7 +25,7 @@ platform_properties:
         [
           {"dependency": "curl"}
         ]
-      os: Debian
+      os: Ubuntu
```

Comment:
With platform value, we do not need target ones.

Reply:
Removed them. thanks.